cfgs/local.toml

```diff
@@ -14,7 +14,7 @@ key = ""
 cert = ""
 
 [signal.auth]
-enabled = true
+enabled = false 
```

Comment:
false by default?

Reply:
this is the local dev config file so I think its fine :) helm generates the toml file from a template with params when running in k8s